backend/src/database/collectionsUtils/studentLogUtils.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+/* eslint-disable func-names */
+
+import { StudentLog } from '../models/studentLog';
+import { User } from '../models/user';
+
+export const addStudentLog = async (logData) => {
+  return new StudentLog(logData).save();
+};
+
+export const logUserJoin = async (studentId, sessionId) => {
+  User.findOne({ _id: studentId, role: 'student' }, function (err) {
```

Comment:
```suggestion
  User.findOne({ _id: studentId, role: 'student' }, (err) => {
```
kosmetyka, ale jeżeli w callbacku nie odwołujemy sie do schemy bezpośrednio przez słowo kluczowe ```this``` to chyba nie powinno być problemów z funkcja strzałkową